test/access dict.py

```diff
@@ -0,0 +1,11 @@
+Employee = {"Name": "MYNK", "Age": 18, "salary":25000000}

```

Comment:
CAN YOU TELL ME WHAT IS KEY AND VALUES IN THIS DICTIONARY?

Reply:
Name, age and salary are keys and mynk, 18 and 25000000 are the values in this dictionary respectively